app/models/inventory.rb

```diff
@@ -48,4 +50,10 @@ def owner?(user:)
   def member?(user:)
     self.members.where(user: user).exists?
   end
+
+  private
+  def add_facilitator_owner
+    return unless owner
+    facilitators.create(user: owner)
```

Comment:
I'm thinking this is fine, but two questions come up:

 - How do we guard against repeated entries of the same owner?  Anecdotally I can't add seem to add the same owner as a facilitator, but I don't see a test covering this scenario.

 - I noticed that the `invited_at` date is `nil` for a facilitator created in this way.  Ideally, would we want to say that they were invited at the same time this was created (or `Time.now`)?